### DISCUSSION TITLE
Welcome to Pico-RGB-Matrix-Clock Discussions!

### DISCUSSION THREAD

**Discussion by jake1164** [jake1164/Pico-RGB-Matrix-Weather-Clock#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rvabrady** [comment]
HELP!! I copied the code to my PI PICO W but nothing is displayed! I am using Circuit Python and a Waveshare 64x32 Matrix. I believe this is the same hardware as your using? However, I am VERY new to python and LED matrix's. Can you tell me how to get it to work? It would be appreciated.    james.brady7070@gmail.com
